source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+
+.. code-block:: console
+   
+   Created symlink /etc/systemd/system/multi-user.target.wants/postgresql.service → /usr/lib/systemd/system/postgresql.service.
+   Cloning into 'telemetrics-backend'...
+   remote: Counting objects: 344, done.
+   remote: Compressing objects: 100% (53/53), done.
+   remote: Total 344 (delta 30), reused 50 (delta 20), pack-reused 268
+   Receiving objects: 100% (344/344), 130.20 KiB | 1.40 MiB/s, done.
+   Resolving deltas: 100% (177/177), done.
+   '/tmp/telemetrics-backend/scripts/collector_uwsgi.ini' -> '/tmp/telemetrics-backend/collector/collector_uwsgi.ini'
+   '/tmp/telemetrics-backend/scripts/telemetryui_uwsgi.ini' -> '/tmp/telemetrics-backend/telemetryui/telemetryui_uwsgi.ini'
+   mkdir: created directory '/var/www/telemetry/collector/uwsgi-spool'
+   mkdir: created directory '/var/www/telemetry/telemetryui/uwsgi-spool'
+   '/tmp/telemetrics-backend/scripts/uwsgi.service' -> '/etc/systemd/system/uwsgi.service'
+   mkdir: created directory '/etc/nginx'
+   mkdir: created directory '/etc/nginx/conf.d'
+   '/usr/share/nginx/conf/nginx.conf.example' -> '/etc/nginx/nginx.conf'
+   Created symlink /etc/systemd/system/multi-user.target.wants/nginx.service → /usr/lib/systemd/system/nginx.service.
+   mkdir: created directory '/etc/uwsgi'
+   mkdir: created directory '/etc/uwsgi/vassals'
+   Created symlink /etc/systemd/system/multi-user.target.wants/uwsgi.service → /etc/systemd/system/uwsgi.service.
+   ALTER ROLE
+   sed: can't read /tmp/telemetrics-backend/collector/config.py: No such file or directory
+   cp: cannot stat '/tmp/telemetrics-backend/collector/config.py': No such file or directory
+   sed: can't read /tmp/telemetrics-backend/telemetryui/config.py: No such file or directory
+   cp: cannot stat '/tmp/telemetrics-backend/telemetryui/config.py': No such file or directory
+   Already using interpreter /usr/bin/python3
+   Using base prefix '/usr'
+   New python executable in /var/www/telemetry/venv/bin/python3
+   Not overwriting existing python script /var/www/telemetry/venv/bin/python (you must use /var/www/telemetry/venv/bin/python3)
+   Installing setuptools, pip, wheel...done.
+   INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+   INFO  [alembic.runtime.migration] Will assume transactional DDL.
+   INFO  [alembic.runtime.migration] Running upgrade  -> 3230c615d6e0, empty message
+   INFO  [alembic.runtime.migration] Running upgrade 3230c615d6e0 -> 466cf2f35d67, empty message
+
+   Install complete (installation folder: /var/www/telemetry)
+
+Once the installation is complete you can use your web browser and view the
+new server by opening the web browser on the system you installed the backend
+server onto and type in ``localhost`` in the address bar.  You should see a
+web page similar to the one shown in figure 1:
+
+.. figure:: figures/telemetry-backend-1.png
+   :scale: 50 %
+   :alt: Telemetry UI
+
+   Figure 1: :guilabel:`Telemetry UI`
+
+
+Redirect telemetry records
```

Comment:
I am conflicted about this section's place in the document. I think this section should come after "Test the new ..." section. However, I can see arguments for both. What was your reasoning for placing it before the test?

Reply:
I want to test the new installation but in order to do that I need to first tell the telemd daemon about the new server location before sending the test hprobe record to it, otherwise it will go to the default server.  I can further clarify if that doesn't make sense.

Reply:
It makes sense. I guess then my question would be: Why do we want to send the test record to a custom server instead of the default?

Reply:
to verify that the new server implementation is functional.